lambdas/check-coi/src/test/java/uk/gov/di/ipv/core/checkcoi/CheckCoiHandlerTest.java

```diff
@@ -199,6 +239,14 @@ void shouldReturnPassedForSuccessfulFullNameAndDobCheck()
                 assertEquals(
                         new AuditExtensionCoiCheck(CoiCheckType.FULL_NAME_AND_DOB, true),
                         auditEventsCaptured.get(1).getExtensions());
+
+                var restrictedAuditData =
+                        getRestrictedAuditDataNodeFromEvent(auditEventsCaptured.get(1));
+                assertTrue(restrictedAuditData.has("newName"));
+                assertTrue(restrictedAuditData.has("oldName"));
+                assertTrue(restrictedAuditData.has("newBirthDate"));
+                assertTrue(restrictedAuditData.has("oldBirthDate"));
+                assertTrue(restrictedAuditData.has("device_information"));
```

Comment:
Doing it this way instead of 
```
assertEquals( new AuditRestrictedData(...), auditEventsCaptured.get(1).getRestricted()))
```
as the device information is encoded and so would result in the failed assertion of the above.